docs/maatregelen/menselijke_tussenkomst_eis_in_programma_van_eisen.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Menselijke tussenkomst is een vast onderdeel van het Programma van Eisen bij aanbestedingen
+toelichting: Neem menselijke tussenkomst op als standaard eis in het Programma van Eisen. 
+vereiste:
+- recht_op_niet_geautomatiseerde_besluitvorming
+levenscyclus:
+- ontwikkelen
+- monitoring-en-beheer
+bouwblok:
+- publieke-inkoop
+rollen:
+- behoeftesteller
+- proceseigenaar
+- gebruiker
+- inkoopadviseur
+---
+
+<!-- tags -->
+## Maatregel
+
+Neem menselijke tussenkomst op als standaard eis in het Programma van Eisen.
+
+
+## Toelichting
+
+De eis stellen is nooit tevergeefs, want als achteraf blijkt dat het niet aan de hand is, doet de bepaling ook geen pijn.
+Het moment van daadwerkelijk constateren is bij oplevering, dus is dit veelal geen punt van waardering in een inkoopproces, anders dan als gestelde eis aan het op te leveren product.
+Dit laat onverlet dat het een criterium is die al in een inkooptraject kan worden beoordeeld, zoals het bij het aanschaffen van standaardproducten en/of diensten.
+
+Het lijkt overbodig, maar het zien begint met het kunnen zien en kijken.
```

Comment:
Deze tekst is misschien iets te filosofisch, en ik zie niet helemaal in wat er nu bedoeld wordt.